openpose/get-docker.sh

```diff
@@ -0,0 +1,525 @@
+#!/bin/sh
+set -e
+# Docker CE for Linux installation script
+#
+# See https://docs.docker.com/engine/install/ for the installation steps.
+#
+# This script is meant for quick & easy install via:
+#   $ curl -fsSL https://get.docker.com -o get-docker.sh
```

Comment:
This file should not be here. Just use this curl command in your install script.

Reply:
Removed